service/worker/deletenamespace/workflow.go

```diff
@@ -139,6 +139,16 @@ func validateNamespace(ctx workflow.Context, nsID namespace.ID, nsName namespace
 		return fmt.Errorf("%w: ValidateProtectedNamespacesActivity: %v", errors.ErrUnableToExecuteActivity, err)
 	}
 
+	ctx2 := workflow.WithLocalActivityOptions(ctx, localActivityOptions)
```

Comment:
`GetVersion()`?

Reply:
This is very short lived WF (actual deletion is in separate WF which is started in abandon child mode). I think it is ok not to versioned it (I've already made few changes which breaks compatibility).